src/site/content/en/lighthouse-performance/third-party-facades/index.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: Lazy load third-party resources with facades
+description: |
+  Learn about the opportunities to lazy load third-party resources with facades.
+date: 2020-12-01
+updated: 2020-12-01
```

Comment:
```suggestion
#updated: 2020-12-01
```
I omit this until we actually update the doc because it looks funny IMO

![image](https://user-images.githubusercontent.com/4713486/100799117-6deb7780-33d9-11eb-9720-b41bada24ebe.png)